projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -904,3 +904,18 @@ function get_paywall_simple() {
 <!-- /wp:columns -->
 ';
 }
+
+/**
+ * Determine if the newsletter categories are enabled for the site.
+ *
+ * This function checks whether the blog has the 'newsletter_categories' sticker enabled.
+ * If the sticker is enabled, the function will return true.
+ * If not, it applies the 'jetpack_newsletter_categories' filter and returns the result (default is false).
+ */
+function newsletter_categories_enabled_for_site() {
+	if ( function_exists( 'has_blog_sticker' ) && has_block_sticker( 'newsletter_categories' ) ) {
```

Comment:
```suggestion
	if ( function_exists( 'has_blog_sticker' ) && has_blog_sticker( 'newsletter_categories' ) ) {
```
Might be a typo here.